### DISCUSSION TITLE
Incrementally Adopting

### DISCUSSION THREAD

**Discussion by theguriev** [nuxt/nuxt#16652] [Questions]
Hey guys.
I have a question for **NuxtJS** maintainers or developers who have had similar experiences...

We have an ancient technology application and would like to switch to **Nuxt3**. What is the best way to do it without stopping the project? Have you thought about it?  We can't stop the project and rewrite everything since it is bad for business and I think a lot of companies are in a similar situation. 

I haven't found any description of how to do it properly except **NextJS** https://nextjs.org/docs/migrating/incremental-adoption. Do you have any ideas?

**Comment by TheAlexLichter** [comment]
Have you taken a look at [nuxt bridge](https://github.com/nuxt/bridge/)?

**Comment by theguriev** [comment]
Yep. But it works only if your previous project is written on nuxt 2 or
something similar. In my case it’s different.

**Reply by Lyokolux** [reply]
I have also a different use case: a vue project with a CMS as backend and multiple services.
We plan to migrate everything with Nuxt and Storyblok. I will document the process as we go in November.

**Reply by TheAlexLichter** [reply]
Fair point. About the listed techinques in the blogpost:
* The subpath technique is always an option.
* Thanks to Nitro, you can now also use rewrites (with a little more effort though I'd say)

**Reply by theguriev** [reply]
> I have also a different use case: a vue project with a CMS as backend and multiple services.
We plan to migrate everything with Nuxt and Storyblok. I will document the process as we go in November.


Looking forward to read your story :)
